questionnaires/templates/questionnaires/tags/already_completed.html

```diff
@@ -2,11 +2,11 @@
 
 {# use page type when we have translations for poll and quiz #}
 <p class="survey-page__already-completed">
-    {% if page.get_type == 'survey' %}
+    {% if page.get_type == 'survey' or questionnaire.get_type == 'survey' %}
```

Comment:
you can avoid this `or` condition by passing the same name of the variable, you'll have to update this file's usages, for reference check [here](https://github.com/unicef/iogt/blob/develop/questionnaires/templates/questionnaires/tags/embedded_poll.html#L31)